cmake/thirdparty/FindCUDA/run_nvcc.cmake

```diff
@@ -74,7 +74,7 @@ set(CUDA_NVCC_FLAGS @CUDA_NVCC_FLAGS@ ;; @CUDA_WRAP_OPTION_NVCC_FLAGS@) # list
 @CUDA_NVCC_FLAGS_CONFIG@
 set(nvcc_flags @nvcc_flags@) # list
 set(CUDA_NVCC_INCLUDE_DIRS "@CUDA_NVCC_INCLUDE_DIRS@") # list (needs to be in quotes to handle spaces properly).
-set(CUDA_NVCC_COMPILE_DEFINITIONS "@CUDA_NVCC_COMPILE_DEFINITIONS@") # list (needs to be in quotes to handle spaces properly).
+set(CUDA_NVCC_COMPILE_DEFINITIONS @CUDA_NVCC_COMPILE_DEFINITIONS@) # list
```

Comment:
Should this be here?

Reply:
Yup. Quoting CUDA_NVCC_COMPILE_DEFINITIONS breaks defines that include quotes.  I think it's a copy-paste error from the line above, where it makes sense to quote the INCLUDE_DIRS to allow for spaces in paths.

Reply:
Sounds good 😄